shared/actions/json/wallets.json

```diff
@@ -40,11 +40,13 @@
     "createNewAccount": {
       "_description": "Add a new wallet to your account",
       "name": "string",
+      "show?": "boolean",
```

Comment:
Since sometimes we want to navigate to our newly created account (creating a wallet from the wallet list) and other times just navigate up (creating a wallet from the send form), it seems like the best way to handle this is to pass in a prop from `routeProps` then pass that through the actions that handle the navigation.

I'm not a super big fan of the duplicated `show` prop in both the `create` and `created` forms of the new account and link account actions, but idk a better way to handle it